requirements/base.txt

```diff
@@ -2,7 +2,7 @@ Jinja2
 # This should be changed to msgpack-python for Packages
 # msgpack-python>0.3,!=0.5.5
 msgpack>=0.5,!=0.5.5
-PyYAML<5.1
+PyYAML
```

Comment:
Could we have instead `PyYAML>5.1`?

Reply:
latest on RHEL 8 is less than 5.1 for PyYAML, recall it as 3.11. Latest Fedora has 5.x